src/Controller/StudentController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StudentController extends AbstractController
+{
+    /**
+     * @Route("/student/", name="student")
+     */
+    public function studentAccount(Request $request)
+    {
+        $name = $request->get('name');
+        $project = $request->get('project');
+
+        return $this->render('home/student.html.twig', [
+            'name' => $name,
+            'project' => $project
+        ]);
+    }
+
+    /**
+     * @Route("/students.json", name="students_json")
+     */
+    public function showJson(KernelInterface $kernel)
```

Comment:
Jei padedamas failas į `public` katalogą – tai jis pasiekiamas ir be `PHP` įsikišimo.

`nginx` arba `apache` tokį failą gali grąžinti daug greičiau (ir su mažiau atminties), negu naudojant `json_decode(file_get_contents(`

`PHP` stiprioji pusė yra, kai reikia pateikti dinaminį turinį (pvz. `ar gaus 10 balų`)

P.S. dėl to ir testas lūžta:
![image](https://user-images.githubusercontent.com/1453957/70462718-2848e700-1ac4-11ea-96e1-bfce6800dc7a.png)